README.md

```diff
@@ -666,6 +666,130 @@ const updateContractTransaction: AccountTransaction = {
 
 Finally, to actually update the contract on the chain, send the constructed `updateContractTransaction` to the chain using `sendAccountTransaction`. (See [Send Account Transaction](#Send-Account-Transaction) for how to do this)
 
+## Smartcontract with parameters
```

Comment:
I have some suggestions to the Readme , which I have in this commit:
https://github.com/Concordium/concordium-node-sdk-js/commit/9b928c355eff5380a13f73394d7fe194fe8d1c42
You should be able to cherry-pick it.